index.bs

```diff
@@ -9479,6 +9490,7 @@ script=].
         functionDeclaration: text,
         ? arguments: [*script.ChannelValue],
         ? contexts: [+browsingContext.BrowsingContext],
+        ? applyToOpeningContexts: bool .default false,
```

Comment:
My initial bikeshedding feedback is that I don't love the name `applyToOpeningContexts` because it's long and yet misleading; I don't see how auxiliary browsing contexts (in HTML spec terms) can be considered `opening`. `inherit: true` might be fine, or `addToOpenedContexts` if we want something longer and less ambiguous.

Reply:
I am fine with `inherit`although I imagine at some point there could multiple ways to inherit configurations. The `opening` idea comes from the fact that contexts (in BiDi) have an opener ID so something is "opening" them at the time we apply this configuration. "Opened" makes it sound to me like the configuration will apply after the context is "opened" but I think this is not important.

Reply:
Updated to inherit for now.